### PR TITLE
PGB-1238 Add partners embedded relation to BRP ingeschrevenpersonen

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: src/gobstuf/stuf/brp/response_mapping.py [hunk 1]
```diff
@@ -9,6 +9,8 @@ class Mapping(ABC):
 
     Provides a filter method to filter out attributes and/or objects
     """
+    related = {}
+    is_relation = False
```
Comment (Vegter) [inline src/gobstuf/stuf/brp/response_mapping.py:14]: Do all Mappings share the same related and is_relation?
It is now a class variable that is shared by all subclasses. I don't know if this is the intention

File: src/gobstuf/rest/brp/base_view.py [hunk 2]
```diff
@@ -302,6 +302,20 @@ def _build_response(self, response_obj: StufMappedResponse, **kwargs):
             }
         }, {})
 
+    def _transform_query_parameter_value(self, value: str):
+        if not value:
```
Comment (Vegter) [inline src/gobstuf/rest/brp/base_view.py:306]: Just a general remark, can you add docstrings for the methods?

Comment (JKoetsier) [reply src/gobstuf/rest/brp/base_view.py:306]: You're a bit too quick. Was already on it, saw that I forgot some important ones

Comment (Vegter) [reply src/gobstuf/rest/brp/base_view.py:306]: Thanks!

**Pull request merged**